prelude/README.md

```diff
@@ -61,8 +61,7 @@ element `id`, just composable morphisms.
 
 `Semigroupoid`s should obey the following rule:
 
-Association:
-    `forall p q r. p <<< (q <<< r) = (p <<< q) <<< r`
+- Association: `forall p q r. p <<< (q <<< r) = (p <<< q) <<< r`
```

Comment:
Isn't this more commonly referred to as associativity? That's certainly how I've always heard it and how I use it myself.